tests/dags/maintenance/test_pr_review_reminders.py

```diff
@@ -78,23 +87,32 @@ def post_comment(*args, **kwargs):
         body = args[3]
         posted_comments[pr_number].append(body)
 
-    monkeypatch.setattr(
-        "openverse_catalog.dags.maintenance.pr_review_reminders.pr_review_reminders.GitHubAPI.get_open_prs",
-        get_prs,
-    )
-    monkeypatch.setattr(
-        "openverse_catalog.dags.maintenance.pr_review_reminders.pr_review_reminders.GitHubAPI.get_issue_comments",
-        get_comments,
-    )
-    monkeypatch.setattr(
-        "openverse_catalog.dags.maintenance.pr_review_reminders.pr_review_reminders.GitHubAPI.post_issue_comment",
-        post_comment,
-    )
+    def delete_comment(*args, **kwargs):
+        comment_number = args[2]
+        deleted_comments.append(comment_number)
+
+    def get_reviews(*args, **kwargs):
+        pr_number = args[2]
+        return pull_reviews[pr_number]
+
+    def patch_gh_fn(fn, impl):
+        monkeypatch.setattr(
+            f"openverse_catalog.dags.maintenance.pr_review_reminders.pr_review_reminders.GitHubAPI.{fn}",
+            impl,
+        )
+
+    patch_gh_fn("get_open_prs", get_prs)
+    patch_gh_fn("get_issue_comments", get_comments)
+    patch_gh_fn("post_issue_comment", post_comment)
+    patch_gh_fn("delete_issue_comment", delete_comment)
+    patch_gh_fn("get_pull_reviews", get_reviews)
 
     yield {
         "pulls": pulls,
         "pull_comments": pull_comments,
         "posted_comments": posted_comments,
+        "deleted_comments": deleted_comments,
```

Comment:
It doesn't look like this is referenced in any tests, should we add some assertions to make sure intended comments are deleted?

Reply:
It is used here: https://github.com/WordPress/openverse-catalog/blob/bd1a26537c4c4be8e019452b722eb1d93b878438/tests/dags/maintenance/test_pr_review_reminders.py#L200

Reply:
Ah, sorry I missed that. Thanks!